doc/api/cli.md

```diff
@@ -178,6 +178,16 @@ added: v8.0.0
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
 
+## `--abort-on-uncaught-exception`
+<!-- YAML
+added: v0.10
+-->
+
+Abort (dump core) when an uncaught exception is thrown. On most systems this will
+generate a core file, though `ulimit -c unlimited` may be necessary if resource
```

Comment:
[just asking] Are the clauses about `ulimit` and on how to use core-dumps needed? Can't we assume (or link to) prior knowledge?